MvvmCross.Forms/Core/IMvxFormsSetupHelper.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MS-PL license.
+// See the LICENSE file in the project root for more information.
+
+using MvvmCross.Binding.BindingContext;
+using MvvmCross.Binding.Bindings.Target.Construction;
+using MvvmCross.Forms.Presenters;
+using MvvmCross.Presenters;
+using Xamarin.Forms;
+
+namespace MvvmCross.Forms.Core
+{
+    public interface IMvxFormsSetupHelper
```

Comment:
How about naming this new guy something like `IMvxFormsAgnosticSetup`? `Helper` sounds too general

Reply:
Hmm perhaps, although I feel that Helper is actually appropriate here since the class that implements this interface is not actually a setup class, it's a class designed to do some of the setup actions on behalf of the actual setup class (ie a helper). I think AgnosticSetup still implies that it is a setup class
Other options?